doc/src/guides/solving/solve-ode.md

```diff
@@ -0,0 +1,467 @@
+{meth}`~sympy.core.basic.Basic.subs`:
+
+```py
+>>> from sympy import Function, dsolve, Derivative, symbols, pi
+>>> x, C1, C2 = symbols("x, C1, C2")
+>>> y = Function('y')
+>>> result = dsolve(Derivative(y(x), x, x) + 9*y(x), y(x)).rhs
+>>> result
+C1*sin(3*x) + C2*cos(3*x)
+>>> result.subs({C1: 7, C2: pi})
+7*sin(3*x) + pi*cos(3*x)
+```
+
+### Numerically Solve an ODE in SciPy
+
+A common workflow which leverages
+[SciPy's](https://docs.scipy.org/doc/scipy/index.html) fast numerical ODE
+solving is
+1. set up an ODE in SymPy
+2. convert it to a numerical function using {func}`~.lambdify`
+3. solve the initial value problem by numerically integrating the ODE using
+   SciPy's `solve_ivp`.
+
+```{warning}
+{func}`~.lambdify` uses {external:func}`~.exec` to dynamically execute Python code, and thus should not be used on unsanitized input.
+```
+
+Here is an [example from the field of chemical
+kinetics](https://www.sympy.org/scipy-2017-codegen-tutorial/notebooks/25-chemical-kinetics-intro.html)
+where the nonlinear ordinary differential equations take this form:
+
+$$ r_f = & k_f y_0(t)^2  y_1(t) \\
+r_b = & k_b y_2(t)^2 \\
+\frac{d y_0(t)}{dt} = & 2(r_b - r_f) \\
+\frac{d y_1(t)}{dt} = & r_b - r_f \\
+\frac{d y_2(t)}{dt} = & 2(r_f - r_b) $$
+
+and
+
+$$
+
+\vec{y}(t) = \begin{bmatrix} y_0(t) \\
+y_1(t) \\
+y_2(t) \end{bmatrix} $$
+
+```{eval-rst}
+.. plot:: 
+  :format: doctest 
+  :include-source: True
+
+  >>> from sympy import symbols, lambdify
+  >>> import numpy as np
+  >>> import scipy.integrate
+  >>> import matplotlib.pyplot as plt
+  >>> def rhs(t, y, kf, kb):
+  ...     rf = kf * y[0]**2 * y[1]
+  ...     rb = kb * y[2]**2
+  ...     return [2*(rb - rf), rb - rf, 2*(rf - rb)]
+  >>> y, (kf, kb) = symbols('y:3'), symbols('kf kb')
+  >>> ydot = rhs(None, y, kf, kb) # derivative of the function y(t); value given by function rhs (right-hand side) for input values y, kf, and kb
+  >>> ydot
+  [2*kb*y2**2 - 2*kf*y0**2*y1, kb*y2**2 - kf*y0**2*y1, -2*kb*y2**2 + 2*kf*y0**2*y1]
+  >>> t = symbols('t') # not used in this case
+  >>> f = lambdify((t, y, kf, kb), ydot) # convert the SymPy symbolic expression for ydot into a form that SciPy can evaluate numerically, f
+  >>> k_vals = np.array([0.42, 0.17]) # arbitrary in this case
+  >>> y0 = [1, 1, 0] # initial condition (initial values)
+  >>> t_eval = np.linspace(0, 10, 50) # evaluate integral from t = 0-10 for 50 points
+  >>> solution = scipy.integrate.solve_ivp(f, (0, 10), y0, t_eval=t_eval, args=k_vals) # call SciPy's solve_ivp by passing it the function f, the interval of integration, the initial state, and the arguments to pass to the function f
+  >>> solution
+      {'message': 'The solver successfully reached the end of the integration interval.', 'nfev': 68, 'njev': 0, 'nlu': 0, 'sol': None, 'status': 0, 'success': True, 't': [ 0.          0.20408163  0.40816327  0.6122449   0.81632653  1.02040816
+        1.2244898   1.42857143  1.63265306  1.83673469  2.04081633  2.24489796
```

Comment:
You can replace unimportant chunks of the output with `...`

Reply:
Thanks. I plan on taking that one step further: Assuming the plot will work, I don't think it's even necessary to include the solution output at all.